### ISSUE TITLE
Error completing request.

### ISSUE THREAD

**Issue by CCpt5** [LonicaMewinsky/frame2frame#1]
Hello, I know you're just starting on this (I found it on a github search by recently updated Automatic1111 extensions). But due to that, I figured maybe you're looking for feedback. I installed using the URL, put a 15sec mp4 in the box (loaded fine), added a prompt, but got the following error when clicking generate. It could very well be my setup or something wrong w/ my personal installation/extensions/etc. I have had occasional crashes the past few weeks. Good luck and thanks for continuing to push the bar w/ SD via your work.

===
Restarting UI...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Loading weights [ca13540e36] from e:\Stable Diffusion Checkpoints\2023-03-04 - Topnotch 66 - (Flip half) - 40img - (Flip half) - 3000 steps (of 5000 max).ckpt
Loading VAE weights specified in settings: C:\stable-diffusion-webui\models\VAE\vae-ft-mse-840000-ema-pruned.ckpt
Applying xformers cross attention optimization.
Weights loaded in 1.2s (load weights from disk: 0.7s, apply weights to model: 0.1s, load VAE: 0.1s, move model to device: 0.2s).
Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\proactor_events.py", line 162, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host
Will process 1 animation(s) with 3430 total generations.
INFO:sd_dynamic_prompts.dynamic_prompting:Prompt matrix will create 1 images in a total of 1 batches.
100%|██████████████████████████████████████████████████████████████████████████████████| 28/28 [00:09<00:00,  2.85it/s]
Error completing request                                                          | 28/96040 [00:09<9:17:00,  2.87it/s]
Arguments: ('task(mzs0ni3fgdfr7pk)', 0, 'topnotch artstyle ', '', [], <PIL.Image.Image image mode=RGBA size=1920x1080 at 0x222C1B27A30>, None, {'image': <PIL.Image.Image image mode=RGBA size=1920x1080 at 0x222C1B25FF0>, 'mask': <PIL.PngImagePlugin.PngImageFile image mode=RGBA size=1920x1080 at 0x222C1B278E0>}, None, None, None, None, 36, 0, 4, 0, 1, False, False, 1, 1, 7, 1.5, 0.75, -1.0, -1.0, 0, 0, 0, False, 1080, 1920, 0, 0, 32, 0, '', '', '', [], 11, False, '', 0, True, False, 1, False, False, False, 1.1, 1.5, 100, 0.7, False, False, True, False, False, 0, 'Gustavosta/MagicPrompt-Stable-Diffusion', '', False, 7, 100, 'Constant', 0, 'Constant', 0, 4, True, True, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, '<ul>\n<li><code>CFG Scale</code> should be 2 or lower.</li>\n</ul>\n', True, True, '', '', True, 50, True, 1, 0, False, 4, 1, 'None', '<p style="margin-bottom:0.75em">Recommended settings: Sampling Steps: 80-100, Sampler: Euler a, Denoising strength: 0.8</p>', 128, 8, ['left', 'right', 'up', 'down'], 1, 0.05, 128, 4, 0, ['left', 'right', 'up', 'down'], False, False, 'positive', 'comma', 0, False, False, '', '<p style="margin-bottom:0.75em">Will upscale the image by the selected scale factor; use width and height sliders to set tile size</p>', 64, 0, 2, 1, '', 0, '', 0, '', True, False, False, False, 0, 'Blur First V1', 0.25, 10, 10, 10, 10, 1, False, '', '', 0.5, 1, False, <tempfile._TemporaryFileWrapper object at 0x00000222C1B264A0>, True, True, True, '29', '3430', True, True, True, 0, 0.1, 1, 'None', False, 0, 2, 512, 512, False, None, None, None, 50, False, 4.0, '', 10.0, 'Linear', 3, False, True, 30.0, True, False, False, 0, 0.0, 'Lanczos', 1, True, 0, 0, '<p style="margin-bottom:0.75em">Will upscale the image depending on the selected target size type</p>', 512, 0, 8, 32, 64, 0.35, 32, 0, True, 0, False, 8, 0, 0, 2048, 2048, 2) {}
Traceback (most recent call last):
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 37, in f
    res = func(*args, **kwargs)
  File "C:\stable-diffusion-webui\modules\img2img.py", line 169, in img2img
    processed = modules.scripts.scripts_img2img.run(p, *args)
  File "C:\stable-diffusion-webui\modules\scripts.py", line 376, in run
    processed = script.run(p, *script_args)
  File "C:\stable-diffusion-webui\extensions\frame2frame\scripts\frame2frame.py", line 267, in run
    out_clip.write_videofile(out_filename, codec='h264', progress_bar=False)
TypeError: VideoClip.write_videofile() got an unexpected keyword argument 'progress_bar'

ERROR:asyncio:Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\proactor_events.py", line 162, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host
Will process 1 animation(s) with 459 total generations.
INFO:sd_dynamic_prompts.dynamic_prompting:Prompt matrix will create 1 images in a total of 1 batches.
100%|██████████████████████████████████████████████████████████████████████████████████| 28/28 [00:02<00:00, 13.42it/s]
Error completing request                                                          | 55/96040 [00:55<3:42:01,  7.21it/s]
Arguments: ('task(xjtsx4lighawizg)', 0, 'topnotch artstyle ', '', [], <PIL.Image.Image image mode=RGBA size=960x720 at 0x222C178CFA0>, None, {'image': <PIL.Image.Image image mode=RGBA size=960x720 at 0x222C178F7C0>, 'mask': <PIL.PngImagePlugin.PngImageFile image mode=RGBA size=960x720 at 0x222C178F910>}, None, None, None, None, 36, 0, 4, 0, 1, False, False, 1, 1, 7, 1.5, 0.75, -1.0, -1.0, 0, 0, 0, False, 720, 960, 0, 0, 32, 0, '', '', '', [], 11, False, '', 0, True, False, 1, False, False, False, 1.1, 1.5, 100, 0.7, False, False, True, False, False, 0, 'Gustavosta/MagicPrompt-Stable-Diffusion', '', False, 7, 100, 'Constant', 0, 'Constant', 0, 4, True, True, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, '<ul>\n<li><code>CFG Scale</code> should be 2 or lower.</li>\n</ul>\n', True, True, '', '', True, 50, True, 1, 0, False, 4, 1, 'None', '<p style="margin-bottom:0.75em">Recommended settings: Sampling Steps: 80-100, Sampler: Euler a, Denoising strength: 0.8</p>', 128, 8, ['left', 'right', 'up', 'down'], 1, 0.05, 128, 4, 0, ['left', 'right', 'up', 'down'], False, False, 'positive', 'comma', 0, False, False, '', '<p style="margin-bottom:0.75em">Will upscale the image by the selected scale factor; use width and height sliders to set tile size</p>', 64, 0, 2, 1, '', 0, '', 0, '', True, False, False, False, 0, 'Blur First V1', 0.25, 10, 10, 10, 10, 1, False, '', '', 0.5, 1, False, <tempfile._TemporaryFileWrapper object at 0x00000222C178F070>, True, True, True, '29', '459', True, True, True, 0, 0.1, 1, 'None', False, 0, 2, 512, 512, False, None, None, None, 50, False, 4.0, '', 10.0, 'Linear', 3, False, True, 30.0, True, False, False, 0, 0.0, 'Lanczos', 1, True, 0, 0, '<p style="margin-bottom:0.75em">Will upscale the image depending on the selected target size type</p>', 512, 0, 8, 32, 64, 0.35, 32, 0, True, 0, False, 8, 0, 0, 2048, 2048, 2) {}
Traceback (most recent call last):
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 37, in f
    res = func(*args, **kwargs)
  File "C:\stable-diffusion-webui\modules\img2img.py", line 169, in img2img
    processed = modules.scripts.scripts_img2img.run(p, *args)
  File "C:\stable-diffusion-webui\modules\scripts.py", line 376, in run
    processed = script.run(p, *script_args)
  File "C:\stable-diffusion-webui\extensions\frame2frame\scripts\frame2frame.py", line 267, in run
    out_clip.write_videofile(out_filename, codec='h264', progress_bar=False)
TypeError: VideoClip.write_videofile() got an unexpected keyword argument 'progress_bar'

**Comment by LonicaMewinsky** [comment]
I greatly appreciate the feedback. Have you tried restarting webui entirely since installing? Looks like the progress bar library (tqdm) is the culprit, which is weird. Investigating. :)

**Comment by CCpt5** [comment]
Looks like I may have a problem (or am missing) opencv. Again, it could just be something stupid on my setup. But this is what I'm getting on startup:  

----
venv "C:\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: 0cc0ee1bcb4c24a8c9715f66cede06601bfc00c8
Installing requirements for Web UI

Error running install.py for extension extensions\frame2frame.
Command: "C:\stable-diffusion-webui\venv\Scripts\python.exe" "extensions\frame2frame\install.py"
Error code: 1
stdout: Installing frame2frame requirement: opencv

stderr: Traceback (most recent call last):
  File "C:\stable-diffusion-webui\extensions\frame2frame\install.py", line 10, in <module>
    launch.run_pip(f"install {lib}", f"frame2frame requirement: {lib}")
  File "C:\stable-diffusion-webui\launch.py", line 137, in run_pip
    return run(f'"{python}" -m pip {args} --prefer-binary{index_url_line}', desc=f"Installing {desc}", errdesc=f"Couldn't install {desc}")
  File "C:\stable-diffusion-webui\launch.py", line 105, in run
    raise RuntimeError(message)
RuntimeError: Couldn't install frame2frame requirement: opencv.
Command: "C:\stable-diffusion-webui\venv\Scripts\python.exe" -m pip install opencv --prefer-binary
Error code: 1
stdout: <empty>
stderr: ERROR: Could not find a version that satisfies the requirement opencv (from versions: none)
ERROR: No matching distribution found for opencv

[notice] A new release of pip available: 22.2.1 -> 23.0.1
[notice] To update, run: C:\stable-diffusion-webui\venv\Scripts\python.exe -m pip install --upgrade pip

**Comment by CCpt5** [comment]
This is the bottom line of my Automatic1111 install for version info:

python: 3.10.6  •  torch: 1.13.1+cu117  •  xformers: 0.0.16rc425  •  gradio: 3.16.2  •  commit: [0cc0ee1b]

**Comment by LonicaMewinsky** [comment]
Thanks a bunch. Can you please re-install the extension and restart webui? I changed the requirements list a tad.

**Comment by CCpt5** [comment]
Ok, I'll do that - I did update/restart before seeing your last msg,  and looks like I got past the opencv issue. Still getting that progress_Bar error. I tried toggling anything w/ progress bar in settings, but nothing seemed to fix it.

---

losing server running on port: 7860                                                | 48/9120 [02:29<2:36:00,  1.03s/it]
Restarting UI...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Will process 1 animation(s) with 332 total generations.
INFO:sd_dynamic_prompts.dynamic_prompting:Prompt matrix will create 1 images in a total of 1 batches.
100%|██████████████████████████████████████████████████████████████████████████████████| 16/16 [00:01<00:00, 13.46it/s]
Error completing request                                                           | 63/9120 [05:10<5:48:42,  2.31s/it]
Arguments: ('task(vbyv5gsnguf0lok)', 0, '', '', [], <PIL.Image.Image image mode=RGBA size=960x720 at 0x195533B4580>, None, {'image': <PIL.Image.Image image mode=RGBA size=960x720 at 0x195533B63E0>, 'mask': <PIL.PngImagePlugin.PngImageFile image mode=RGBA size=960x720 at 0x195533B6320>}, None, None, None, None, 20, 0, 4, 0, 1, False, False, 1, 1, 7, 1.5, 0.75, -1.0, -1.0, 0, 0, 0, False, 720, 960, 0, 0, 32, 0, '', '', '', [], 12, False, '', 0, True, False, 1, False, False, False, 1.1, 1.5, 100, 0.7, False, False, True, False, False, 0, 'Gustavosta/MagicPrompt-Stable-Diffusion', '', False, 7, 100, 'Constant', 0, 'Constant', 0, 4, True, True, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, '<ul>\n<li><code>CFG Scale</code> should be 2 or lower.</li>\n</ul>\n', True, True, '', '', True, 50, True, 1, 0, False, 4, 1, 'None', '<p style="margin-bottom:0.75em">Recommended settings: Sampling Steps: 80-100, Sampler: Euler a, Denoising strength: 0.8</p>', 128, 8, ['left', 'right', 'up', 'down'], 1, 0.05, 128, 4, 0, ['left', 'right', 'up', 'down'], False, False, 'positive', 'comma', 0, False, False, '', '<p style="margin-bottom:0.75em">Will upscale the image by the selected scale factor; use width and height sliders to set tile size</p>', 64, 0, 2, 1, '', 0, '', 0, '', True, False, False, False, 0, 'Blur First V1', 0.25, 10, 10, 10, 10, 1, False, '', '', 0.5, 1, False, '', 'Normal', 1, True, 1, 1, 'None', False, False, 'YuNet', 1024, 0.5, 1.5, False, 'face close up,', 0.5, 0.5, False, True, <tempfile._TemporaryFileWrapper object at 0x00000195533A9C60>, True, True, True, '29', '332', True, True, True, 0, 0.1, 1, 'None', False, 0, 2, 512, 512, False, None, None, None, 50, False, 4.0, '', 10.0, 'Linear', 3, False, True, 30.0, True, False, False, 0, 0.0, 'Lanczos', 1, True, 0, 0, 0.0001, '<p style="margin-bottom:0.75em">Will upscale the image depending on the selected target size type</p>', 512, 0, 8, 32, 64, 0.35, 32, 0, True, 0, False, 8, 0, 0, 2048, 2048, 2) {}
Traceback (most recent call last):
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 37, in f
    res = func(*args, **kwargs)
  File "C:\stable-diffusion-webui\modules\img2img.py", line 169, in img2img
    processed = modules.scripts.scripts_img2img.run(p, *args)
  File "C:\stable-diffusion-webui\modules\scripts.py", line 376, in run
    processed = script.run(p, *script_args)
  File "C:\stable-diffusion-webui\extensions\frame2frame\scripts\frame2frame.py", line 267, in run
    out_clip.write_videofile(out_filename, codec='h264', progress_bar=False)
TypeError: VideoClip.write_videofile() got an unexpected keyword argument 'progress_bar'

**Comment by CCpt5** [comment]
So I just noticed that when dropping a video into the box I'm getting an error also, although it shows up and plays fine. Populates the frames info etc:

----
ERROR:asyncio:Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\proactor_events.py", line 162, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host
ERROR:asyncio:Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\proactor_events.py", line 162, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host

**Comment by LonicaMewinsky** [comment]
Looks like there was a dependency missing. I've made a change to the requirements list again -- I hate to ask, since you've already been a great help, but could you please reinstall and try again?

**Comment by CCpt5** [comment]
Still no go. First installed / restarted ui, but also deleted the folder and repulled it. I noticed an error before I think when I first moved an mp4 into the box, not sure if it's related (both):

Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\proactor_events.py", line 162, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host

______

Will process 1 animation(s) with 940 total generations.
INFO:sd_dynamic_prompts.dynamic_prompting:Prompt matrix will create 1 images in a total of 1 batches.
100%|██████████████████████████████████████████████████████████████████████████████████| 16/16 [00:05<00:00,  2.90it/s]
Error completing request                                                          | 16/15040 [00:05<1:26:15,  2.90it/s]
Arguments: ('task(kgd04rl6ebyu0kc)', 0, 'smb artstyle ', '', [], <PIL.Image.Image image mode=RGBA size=1920x1080 at 0x1FA394BEB90>, None, {'image': <PIL.Image.Image image mode=RGBA size=1920x1080 at 0x1FA394BE7D0>, 'mask': <PIL.PngImagePlugin.PngImageFile image mode=RGBA size=1920x1080 at 0x1FA394BE9E0>}, None, None, None, None, 20, 0, 4, 0, 1, False, False, 1, 1, 7, 1.5, 0.75, -1.0, -1.0, 0, 0, 0, False, 1080, 1920, 0, 0, 32, 0, '', '', '', [], 12, False, '', 0, True, False, 1, False, False, False, 1.1, 1.5, 100, 0.7, False, False, True, False, False, 0, 'Gustavosta/MagicPrompt-Stable-Diffusion', '', False, 7, 100, 'Constant', 0, 'Constant', 0, 4, True, True, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, False, 'none', 'None', 1, None, False, 'Scale to Fit (Inner Fit)', False, False, 64, 64, 64, 0, 1, False, '<ul>\n<li><code>CFG Scale</code> should be 2 or lower.</li>\n</ul>\n', True, True, '', '', True, 50, True, 1, 0, False, 4, 1, 'None', '<p style="margin-bottom:0.75em">Recommended settings: Sampling Steps: 80-100, Sampler: Euler a, Denoising strength: 0.8</p>', 128, 8, ['left', 'right', 'up', 'down'], 1, 0.05, 128, 4, 0, ['left', 'right', 'up', 'down'], False, False, 'positive', 'comma', 0, False, False, '', '<p style="margin-bottom:0.75em">Will upscale the image by the selected scale factor; use width and height sliders to set tile size</p>', 64, 0, 2, 1, '', 0, '', 0, '', True, False, False, False, 0, 'Blur First V1', 0.25, 10, 10, 10, 10, 1, False, '', '', 0.5, 1, False, '', 'Normal', 1, True, 1, 1, 'None', False, False, 'YuNet', 1024, 0.5, 1.5, False, 'face close up,', 0.5, 0.5, False, True, <tempfile._TemporaryFileWrapper object at 0x000001FA30935AB0>, True, True, True, '29', '940', True, True, True, 0, 0.1, 1, 'None', False, 0, 2, 512, 512, False, None, None, None, 50, False, 4.0, '', 10.0, 'Linear', 3, False, True, 30.0, True, False, False, 0, 0.0, 'Lanczos', 1, True, 0, 0, 0.0001, '<p style="margin-bottom:0.75em">Will upscale the image depending on the selected target size type</p>', 512, 0, 8, 32, 64, 0.35, 32, 0, True, 0, False, 8, 0, 0, 2048, 2048, 2) {}
Traceback (most recent call last):
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 56, in f
    res = list(func(*args, **kwargs))
  File "C:\stable-diffusion-webui\modules\call_queue.py", line 37, in f
    res = func(*args, **kwargs)
  File "C:\stable-diffusion-webui\modules\img2img.py", line 169, in img2img
    processed = modules.scripts.scripts_img2img.run(p, *args)
  File "C:\stable-diffusion-webui\modules\scripts.py", line 376, in run
    processed = script.run(p, *script_args)
  File "C:\stable-diffusion-webui\extensions\frame2frame\scripts\frame2frame.py", line 267, in run
    out_clip.write_videofile(out_filename, codec='h264', progress_bar=False)
TypeError: VideoClip.write_videofile() got an unexpected keyword argument 'progress_bar'

**Comment by LonicaMewinsky** [comment]
Just to be safe I built a new a1111 dir entirely; I can't seem to recreate the problem. So weird.

I've updated again to run without any arguments (including skipping progress bar). At your convenience can you please try again?

**Comment by CCpt5** [comment]
Sorry that it took me so long to come back around. Buuut looks like your latest version is working!  Rolling through the video and I can see the previews. 

I did still get an error when dropping a video in the box, but so far it doesn't seem to have affected it from running (and looked fine in the video player preview:.  This is my full console log from launch, install, drop video, error:  
 

venv "C:\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: 0cc0ee1bcb4c24a8c9715f66cede06601bfc00c8
Installing requirements for Web UI
Launching Web UI with arguments: --xformers --api --ckpt-dir e:\Stable Diffusion Checkpoints
Loading weights [396c3fd2fa] from e:\Stable Diffusion Checkpoints\2022-12-29 - Topnotch 8 - (Topnotch Artstyle) - 36img contd of 7 - 7000 steps - (50img to 4000steps + 36img to 3000).ckpt
Creating model from config: C:\stable-diffusion-webui\configs\v1-inference.yaml
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
Loading VAE weights specified in settings: C:\stable-diffusion-webui\models\VAE\vae-ft-mse-840000-ema-pruned.ckpt
Applying xformers cross attention optimization.
Textual inversion embeddings loaded(0):
Model loaded in 2.3s (load weights from disk: 0.4s, create model: 0.3s, apply weights to model: 0.2s, apply half(): 0.3s, load VAE: 0.1s, move model to device: 0.3s, load textual inversion embeddings: 0.6s).
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Installing frame2frame requirement: imageio_ffmpeg
Installing frame2frame requirement: moviepy
Installing frame2frame requirement: opencv-python

Closing server running on port: 7860
Restarting UI...
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
Exception in callback _ProactorBasePipeTransport._call_connection_lost(None)
handle: <Handle _ProactorBasePipeTransport._call_connection_lost(None)>
Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python310\lib\asyncio\proactor_events.py", line 162, in _call_connection_lost
    self._sock.shutdown(socket.SHUT_RDWR)
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host
